src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -662,7 +737,7 @@ protected CompletableFuture<ExecutionResult> completeValueForEnum(ExecutionConte
         } catch (NonNullableFieldWasNullException e) {
             return exceptionallyCompletedFuture(e);
         }
```

Comment:
notice here we still use CFs - why?  because they can represent exceptions in themselves.

So if the serlisation fails - we returns an exceptional CF otherwise the materialised values